pom.xml

```diff
@@ -45,7 +45,7 @@
     <vertx.version>4.5.4</vertx.version>
 
     <!--Dependency properties-->
-    <edge-common.version>4.6.0</edge-common.version>
+    <edge-common.version>4.7.0-SNAPSHOT</edge-common.version>
```

Comment:
after releasing edge-common we should specify released version here

Reply:
Fixed